app/views/dvr/requestedDetailedValuation.scala.html

```diff
@@ -16,25 +16,55 @@
 
 @import config.ApplicationConfig
 @import utils.Formatters
+@import views.html.includes.assessmentHeading
 
 @this(mainLayout: views.html.mainLayout)
 
-@(submissionId: String, address: String)(implicit request: Request[_], messages: Messages, config: ApplicationConfig)
+@(submissionId: String, address: String,  localAuthorityRef: String, clientDetails: Option[ClientDetails] = None)(implicit request: Request[_], messages: Messages, config: ApplicationConfig)
 
-@mainLayout(pageTitle=Messages("detailedValuationRequested.page.title")) {
+
+@managePropertiesUrl = @{
+    clientDetails match {
+        case Some(client) =>
+            config.dashboardUrl(
+                s"selected-client-properties?clientOrganisationId=${client.organisationId}&clientName=${client.organisationName}")
```

Comment:
string interpolation like this isn't safe for URLs but I think we do this in many other places. I think it will be good to have a test of what happens when a client org name is "foo&clientName=bar" and review the current situation in codebase, just how widely used is string interpolation and fix it - URL-encoded strings or use the `uri-template` library (out of scope for this ticket)